jupyter-covid19/Dockerfile

```diff
@@ -5,15 +5,15 @@ USER root
 # COPY premade notebook
 RUN mkdir /home/jovyan/covid19-notebook
 
-ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/chore/splashPage/covid19-notebooks/splashPage.html /home/jovyan/covid19-notebook/
+ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/welcome.html /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/jhu-summary-overview/COVID-19-JHU_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/kaggle-demographics/kaggle_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/readme.md /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/requirements.txt /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/chicago-seir-forecast/covid19_seir.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/chicago-seir-forecast/seir_diagram.png /home/jovyan/covid19-notebook/
 RUN chmod -R 777 /home/jovyan/covid19-notebook
-RUN touch /home/jovyan/covid19-notebook/splashPage.html
+RUN touch /home/jovyan/covid19-notebook/welcome.html
```

Comment:
```suggestion
RUN touch /home/jovyan/welcome.html
```

Reply:
`welcome.html` is added to the /home/jovyan/covid19-notebook/ folder in ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/1.2.0/covid19-notebooks/welcome.html /home/jovyan/covid19-notebook/`.

Reply:
right, need to change both